firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/internal/ModelFileManager.java

```diff
@@ -64,8 +63,7 @@ public static ModelFileManager getInstance() {
   }
 
   @NonNull
-  public static ModelFileManager getInstance(FirebaseApp app) {
-    Preconditions.checkArgument(app != null, "Null is not a valid value of FirebaseApp.");
+  public static ModelFileManager getInstance(@NonNull FirebaseApp app) {
     return FirebaseApp.getInstance(app.getName()).get(ModelFileManager.class);
```

Comment:
`app.get(...)` should do the trick. See https://github.com/firebase/firebase-android-sdk/blob/ac3276ce7d5f5181b1c419731495eb88a98537af/firebase-segmentation/src/main/java/com/google/firebase/segmentation/FirebaseSegmentation.java#L72 for reference

Reply:
Thanks for the reference, so it should be like `return app.get(ModelFileManager.class);`?